var/spack/repos/builtin/packages/gromacs/package.py

```diff
@@ -48,13 +49,20 @@ class Gromacs(CMakePackage):
             description='The build type to build',
             values=('Debug', 'Release', 'RelWithDebInfo', 'MinSizeRel',
                     'Reference', 'RelWithAssert', 'Profile'))
+    variant('simd', default='AUTO',
+            description='The SIMD instruction set to use',
+            values=('AUTO', 'None', 'SSE2', 'SSE4.1', 'AVX_128_FMA', 'AVX_256',
```

Comment:
Just a comment, not a request. In #9481 (which hopefully will get in soon) we introduce 2 values for multi-valued variants that have a precise semantic meaning: `none` and `auto`. As you are using the same semantic here, I was wondering if it's an issue to make `AUTO` and `None` already lowercase.

Reply:
Sounds good. We loose direct relation to the [gromacs documentation](http://manual.gromacs.org/documentation/2018.4/install-guide/#simd-support), but it should still be clear enough.

Reply:
#9481 is merged now, in case you want to use the new semantic wrappers as well

Reply:
so by making none and auto lowercase I already did this, right?

Reply:
> so by making none and auto lowercase I already did this, right?

Checked the PR right now. Being `simd` a single-valued variant (i.e. you can't ask for `simd=SSE2,SSE4.1`) this is fine.

#9481 adds wrappers to cover cases where any combination of a set of values is fine.